client/src/test/python/TestHttpClient.py

```diff
@@ -50,7 +50,8 @@ def test_get_with_username_password(self):
 
         args, kwargs = client.session.request.call_args
         self.assertEqual(('GET', 'http://foo.bar'), args)
-        self.assertTrue(kwargs['auth'], ('username', 'password'))
+        # What's the correct behavior here?
+        #self.assertTrue(kwargs['auth'], ('username', 'password'))

```

Comment:
I had to comment this out to get the test to pass.  What's the expected behavior here now?  Is this test even useful now?

Reply:
Thanks, this test is no more useful.